types/metadataV8.go

```diff
@@ -102,6 +102,15 @@ func (m *MetadataV8) FindStorageEntryMetadata(module string, fn string) (Storage
 	return nil, fmt.Errorf("module %v not found in metadata", module)
 }
 
+func (m *MetadataV8) ExistsModuleMetadata(module string) bool {
```

Comment:
Since the function signature is same for v7, v8 and possibly higher versions, is it possible to have it as part of base metadata ?

Reply:
Since the modules list is part of each specifc struct, I would still need to either get a list of modules on each module implementation or do what I do here. I rather keep it like is.